translations/fritzing_ru.ts

```diff
@@ -1269,22 +1269,22 @@ Use the file?</source>
     <message>
         <location filename="../src/help/firsttimehelpdialog.cpp" line="43"/>
         <source>First Time Help</source>
-        <translation>Начальная помощь</translation>
+        <translation>Начальная справка</translation>
     </message>
     <message>
         <location filename="../src/help/firsttimehelpdialog.cpp" line="88"/>
         <source>&lt;br/&gt;The &lt;b&gt;Breadboard View&lt;/b&gt; is meant to look like a &lt;i&gt;real-life&lt;/i&gt; breadboard prototype.&lt;br/&gt;&lt;br/&gt;Begin by dragging a part from the Parts Bin, which is over at the top right. Then pull in more parts, connecting them by placing them on the breadboard or clicking on the connectors and dragging wires. The process is similar to how you would arrange things in the physical world. &lt;br/&gt;&lt;br/&gt;After you&apos;re finished creating your sketch in the breadboard view, try the other views. You can switch views by clicking the Tabs at the top of the window. Because different views have different purposes, parts will look different in the other views.</source>
-        <translation>&lt;br/&gt;&lt;b&gt;Вид &quot;Макетной платы&quot; предназначен для просмотра того, как &lt;i&gt;прототип&lt;/i&gt;выглядит в реальности.&lt;br/&gt;&lt;br/&gt; Начните с перетаскивания отдельных компонентов из каталога компонентов, который находится в правом верхнем углу. Затем перетащите остальные компоненты, соединяя их путём размещения на макете или щёлкая на разъёмах и перетаскивая провода. Этот процесс похож на то, как вы располагали бы вещи в физическом мире. &lt;br/&gt;&lt;br/&gt;После завершения создания эскиза в виде макета попробуйте другие виды. Можно переключать представления, щёлкая вкладки в верхней части окна. Поскольку разные виды имеют разное назначение, компоненты будут выглядеть по-разному в различных видах.</translation>
+        <translation>&lt;br/&gt;Режим&lt;b&gt;Представление макетной платы&lt;/b&gt; позволяет показать, как выглядит &lt;i&gt;настоящий&lt;/i&gt; макет.&lt;br/&gt;&lt;br/&gt;Начните с перетаскивания одного компонента из каталога компонентов, который находится в правом верхнем углу. Затем перетащите остальные компоненты, соединяя их путём размещения на макете или щёлкая по разъёмам и перетаскивая провода. Этот процесс похож на то, как вы располагали бы вещи в физическом мире. &lt;br/&gt;&lt;br/&gt;После завершения создания эскиза в режиме макета попробуйте использовать другие предоставления. Можно переключать представления, щёлкая по вкладкам в верхней части окна. Поскольку разные представления имеют разное назначение, компоненты будут выглядеть по-разному в различных видах.</translation>
```

Comment:
Please conserve special codes like `&quot; `

Reply:
OK, I'll fix this.

Reply:
(Though my version would be true to source formatting.)

Reply:
Ah, using different signs would be ok. However I don't know how by heart how those are represented corretly in Qt. Do you have the possibility to generate a build and check if it looks correct?

Reply:
Unfortunately, I don't have the skills to make a build, but from my previous experience I can tell that the «» quotes do look okay in programs built with .ts (.qm) files containing them.

Reply:
Checked the translation in the application (by copying the .qm file to usr/share) and made a few changes, so it looks fine now.
![Screenshot_20191023_170231](https://user-images.githubusercontent.com/53296253/67400898-04365280-f5b7-11e9-9f1f-7472339b108e.png)

Reply:
The «» quotes look okay as well.
![Screenshot_20191023_170716](https://user-images.githubusercontent.com/53296253/67401327-abb38500-f5b7-11e9-87de-a8059daef985.png)